daprdocs/content/en/operations/components/setup-bindings/supported-bindings/_index.md

```diff
@@ -3,30 +3,43 @@ type: docs
 title: "Supported external bindings"
 linkTitle: "Supported bindings"
 weight: 200
-description: The supported external systems that interface with Dapr as input/output bindings
+description: The supported external bindings that interface with Dapr
 no_list: true
 ---
 
+Every binding has its own unique set of properties. Click the name link to see the component YAML for each binding.
+
+
+Table captions:
+
+> `Status`: [Component certification]({{<ref "certification-lifecycle.md">}}) status
+  - [Alpha]({{<ref "certification-lifecycle.md#alpha">}})
+  - [Beta]({{<ref "certification-lifecycle.md#beta">}})
+  - [GA]({{<ref "certification-lifecycle.md#general-availability-ga">}})
+> `Since`: defines from which Dapr Runtime version, the component is in the current status
+
+> `Component version`: defines the version of the component
 ### Generic
 
-| Name | Input<br>Binding | Output<br>Binding | Status |
-|------|:----------------:|:-----------------:|--------|
-| [Apple Push Notifications (APN)]({{< ref apns.md >}}) |  | ✅ | Alpha |
-| [Cron (Scheduler)]({{< ref cron.md >}}) | ✅ | ✅ | Alpha |
-| [HTTP]({{< ref http.md >}})           |    | ✅ | Alpha |
-| [InfluxDB]({{< ref influxdb.md >}})       |    | ✅ | Alpha |
-| [Kafka]({{< ref kafka.md >}})         | ✅ | ✅ | Alpha |
-| [Kubernetes Events]({{< ref "kubernetes-binding.md" >}}) | ✅ |    | Alpha |
-| [MQTT]({{< ref mqtt.md >}})           | ✅ | ✅ | Alpha |
-| [MySQL]({{< ref mysql.md >}})       |    | ✅ | Alpha |
-| [PostgreSql]({{< ref postgres.md >}})       |    | ✅ | Alpha |
-| [Postmark]({{< ref postmark.md >}})       |    | ✅ | Alpha |
-| [RabbitMQ]({{< ref rabbitmq.md >}})   | ✅ | ✅ | Alpha |
-| [Redis]({{< ref redis.md >}})         |    | ✅ | Alpha |
-| [SMTP]({{< ref smtp.md >}})         |    | ✅ | Alpha |
-| [Twilio]({{< ref twilio.md >}})       |    | ✅ | Alpha |
-| [Twitter]({{< ref twitter.md >}})       | ✅ | ✅ | Alpha |
-| [SendGrid]({{< ref sendgrid.md >}})       |    | ✅ | Alpha |
+| Name | Input<br>Binding | Output<br>Binding | Status | Component version | Since |
+|------|:----------------:|:-----------------:|--------|-------- | ---------|
+| [Apple Push Notifications (APN)]({{< ref apns.md >}}) |  | ✅ | Alpha | v1 | 1.0 | 
```

Comment:
Its odd that the Component and Runtime Since version are in different formats. I would change all the components from v1 to be 1.0 so that this is the same as the runtime. Or have v1.0 for all of them (but the v is redundant here)

Reply:
These are versioned independently, actually. And there is no v1.1 component, for example. It is just a coincidence right now because we are starting it.

Reply:
@msfussell There is a reason why the format is different, in the component YAML, for the version field, a component has the version as `v0`,  `v1` etc. I wanted to maintain that format here also.

Reply:
Yes. I got this wrong. This is correct